test/controllers/user.controller.test.js

```diff
@@ -221,10 +221,9 @@ describe('Login Route', () => {
           expect(res).to.have.status(200);
           expect(res.body.success).to.be.eql(true);
           expect(res.body.message).to.be.eql('Login successful');
-          expect(res.body.payload).to.have.property('firstName');
-          expect(res.body.payload).to.have.property('lastName');
-          expect(res.body.payload).to.have.property('email');
-          expect(res.body.payload).to.have.property('token');
+          expect(res.body.user.firstName).to.be.eql(user.firstName);
+          expect(res.body.user.lastName).to.be.eql(user.lastName);
+          expect(res.body.user.email).to.be.eql(user.email);
```

Comment:
Can you return the test for the `token`? We need to confirm that the token is returned when a user login. Not compulsory now, but we might need to update the tests later to confirm that the right token is returned.
```suggestion
          expect(res.body.user.email).to.be.eql(user.email);
          expect(res.body.payload).to.have.property('token');
```